todo_app/app.py

```diff
@@ -21,4 +24,26 @@ def addItem():
         return redirect(url_for('index'))
 
 
+@app.route('/complete', methods=['POST'])
+def completeItem():
+    if request.method == 'POST':
```

Comment:
The route decorator ensures that this condition is always true. There is no need to check again.